app/presenters/flood_risk_engine/reviewing/review_presenter.rb

```diff
@@ -0,0 +1,37 @@
+#
+# Aggregates and exposes tabular summarized enrollment data for use in a view, for example
+# the 'check your answers' page or the confirmation email.
+#
+module FloodRiskEngine
+  module Reviewing
+    class ReviewPresenter
+      attr_reader :options
+
+      def initialize(enrollment, i18n_scope)
+        @options = {
+          i18n_scope: i18n_scope,
+          enrollment: enrollment
+        }
+      end
+
+      # Add to this function as required. Define new rowbuilders for each row
+      # of review data, and make them do the the work.
+      def rows
```

Comment:
Is it worth splitting this into two - one method that returns `rows` and another that populates the row objects. Currently calling rows twice would build the array twice.
```ruby
def rows
  @rows ||= build_rows
end

def build_rows
  <current rows code>
end
```